mappings/net/minecraft/world/GameRules.mapping

```diff
@@ -38,8 +38,9 @@ CLASS net/minecraft/class_1928 net/minecraft/world/GameRules
 	FIELD field_9197 RULE_TYPES Ljava/util/Map;
 	METHOD <init> (Ljava/util/Map;)V
 		ARG 1 rules
-	METHOD method_20744 forEachType (Lnet/minecraft/class_1928$class_4311;)V
-		ARG 0 action
+	METHOD method_20744 visit (Lnet/minecraft/class_1928$class_4311;)V
+		COMMENT Visits all registered game rules using the supplied visitor.
```

Comment:
This needs to be passive form, the visitor is visiting, not GameRules. Something like `Make the visitor visit all game rules.\n\n<p>The visitation involves calling both accept() and acceptX() for every game rule, where X is the current rule's concrete type such as Boolean.`